core/server/worker/src/main/java/alluxio/worker/grpc/DoraWorkerClientServiceHandler.java

```diff
@@ -109,14 +114,20 @@ public StreamObserver<ReadRequest> readBlock(StreamObserver<ReadResponse> respon
   public void getStatus(GetStatusPRequest request,
                         StreamObserver<GetStatusPResponse> responseObserver) {
     try {
-      String ufsFilePath = request.getPath();
-      UfsFileStatus status = mUfsFileStatusCache.get(request.getPath());
+      String alluxioFilePath = request.getPath();
+
+      String ufsFullPath = PathUtils.concatPath(mRootUFS,
+          CommonUtils.stripPrefixIfPresent(alluxioFilePath, mRootUFS));
+      String fn = new AlluxioURI(alluxioFilePath).getName();
+
+      UfsFileStatus status = mUfsFileStatusCache.get(ufsFullPath);
       GetStatusPResponse response = GetStatusPResponse.newBuilder()
           .setFileInfo(
               FileInfo.newBuilder()
-                  .setName(status.getName())
-                  .setPath(ufsFilePath)
-                  .setUfsPath(ufsFilePath)
+                  .setFileId(ufsFullPath.hashCode())
```

Comment:
need change to the new way of producing a hash

Reply:
This field requires a "long" type, and we can not use the new SHA256 hash here.

Also, please note, the "FileId" of a FileInfo is a fake/bogus one, and it will not used by client.
On client, I've also made the changes to use new SHA256 hash as its path in local page cache.